src/openrct2/ride/Ride.cpp

```diff
@@ -2387,7 +2387,10 @@ static void ride_inspection_update(Ride* ride)
 
     int32_t inspectionIntervalMinutes = RideInspectionInterval[ride->inspection_interval];
     if (inspectionIntervalMinutes == 0)
```

Comment:
Could you add a comment clarifying the if gets triggered when the user selected 'Never inspect'?